### DISCUSSION TITLE
How to open overview in synctex mode?

### DISCUSSION THREAD

**Discussion by OopsYao** [ahrm/sioyek#759] [Q&A] [answered]
Hi, I'd like to use overview function in synctex mode, but right clicking opens corresponding tex file, so I cannot open an overview window like before. How do I change this behavior? It would be great if I can change "open tex file" to double clicking in this mode.

It seems like `keyboard_overview` is a workaround, but it gives too many yellow box options to choose, which is hard to read, but all I want is the real links that exist (just like `open_link` does)

**Comment by ahrm** [comment, marked as answer]
You can use `overview_under_cursor`.

**Reply by OopsYao** [reply]
Thanks! Is it possible to support custom keybindings of mouse button?

**Reply by ahrm** [reply]
See https://sioyek-documentation.readthedocs.io/en/latest/configuration.html#shift-click-command-control-click-command-alt-click-command-shift-right-click-command-control-right-click-command-and-alt-right-click-command.

**Reply by OopsYao** [reply]
Oh, I expected it to be in `keys.config`. So currently is it impossible to custom pure right click (without a modifier key) or something like double-click? Using mouse and keyboard at same time can be inconvenient sometimes. 😅

**Reply by ahrm** [reply]
Yes, currently it is impossible. If you use overview more than synctex then you can not use synctex mode and then use `synctex_under_cursor` to do a synctex search.

**Reply by OopsYao** [reply]
Great idea! Thanks~
